### ISSUE TITLE
Possible to separate the lockout from power and pulling the apparatus as different configs? 

### ISSUE THREAD

**Issue by ClemonadeStand** [AntoninJuquel/lethal-company-lockout#3]
I know there is a different function when you take the apparatus vs when power is on or off. I was wondering if you'd be able to add that as a config option for both?

**Comment by ClemonadeStand** [comment]
And also if power being off is the lockdown state would be cool.

**Comment by AntoninJuquel** [comment]
Hello thanks for your feedback.
To be clear,
- You want to have different configuration for taking apparatus vs powering off the facility with the breakerbox ?
_ie: CanPowerOffWithApparatus, CanPowerOffWithBreakerBox_
- And also Power Off -> lockdown
_ie: LockDownWithPower_

**Comment by ClemonadeStand** [comment]
Yes, exactly. Would be awesome. I'm working on a pack that has the breaker box turned off at the start so players have to find that to have lights inside but the door is also locked at the beginning, just a little weird for it to say 'The building is locked down' when I have the main door open when the power turns on.

**Comment by AntoninJuquel** [comment]
1. For _CanPowerOffWithApparatus_ and _CanPowerOffWithBreakerBox_, currently in my mod, I am subscribing to an event that is triggered whenever the power is changed. Therefore, I'll have to investigate how to differentiate the cause.

2. For _LockDownWithPower_, it seems definitely easier.

I also have a suggestion: I could add PowerOffMessage and PowerOnMessage. It could be a lazy workaround to just change the messages when the state changes. 

I'll let you know whenever I make these changes.

Thanks for your interesting requests
